archive/r/rust/fibonacci.rs

```diff
@@ -0,0 +1,37 @@
+const LIMIT: i64 = 92;
+
+fn fibonacci(terms: i64) {
+    if terms > LIMIT {
+        println!("The number of terms you want to calculate is too big!");
+        println!("The limit is 92.");
```

Comment:
Please consider reusing the `LIMIT` variable instead of hardcoding the value here

Reply:
Done